### PR TITLE
Aavruti_Assignment_17_20mins

### PR DESCRIPTION
**Pull request by AavrutiDJun20**
Program 1 : Print count of Digits, Letters, UpperCase, LowerCase and Special Character in given input '1rRpd3F9#K(E'
Program 2: Print First Index and last Index of every char of input 'technocredits'

MethodOfChar

FirstAndLastIndex

### CODE REVIEW

File: src/aavrutiDJun20/FirstAndLastIndexOfChar.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+//Print First Index and last Index of every char of input 'technocredits'
+
+package aavrutiDJun20;
+
+public class FirstAndLastIndexOfChar {
+	
+	void calcFirstAndLastIndex(String input) {
+		for(int index=0;index<input.length();index++) {
+			char ch = input.charAt(index);
+			if(input.indexOf(ch) != input.lastIndexOf(ch) && index == input.indexOf(ch))
```
Comment (KrishnaKTechnocredits) [inline src/aavrutiDJun20/FirstAndLastIndexOfChar.java:10]: Good job.

File: src/aavrutiDJun20/MethodsOfCharacter.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+			char ch = input.charAt(index);
+			if(Character.isDigit(ch) == true) 
+				digitCnt++;
+			else if(Character.isUpperCase(ch) == true) {
+				upperCaseCnt++;
+			    letterCnt++;
+			}
+			else if(Character.isLowerCase(ch) == true) {
+				lowerCaseCnt++;			
+				letterCnt++;
```
Comment (KrishnaKTechnocredits) [inline src/aavrutiDJun20/MethodsOfCharacter.java]: Think in line number 20 and 24 letterCnt this variable is not needed.

Comment (AavrutiDJun20) [reply src/aavrutiDJun20/MethodsOfCharacter.java]: Done.

File: src/aavrutiDJun20/MethodsOfCharacter.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+				digitCnt++;
+			else if(Character.isUpperCase(ch) == true)
+				upperCaseCnt++;
+			else if(Character.isLowerCase(ch) == true)
+				lowerCaseCnt++;
+			else
+				specialChar++;
+		}
+		System.out.println("Total Digit        : " + digitCnt);
+		System.out.println("Total Letters      : " + (upperCaseCnt+lowerCaseCnt));
```
Comment (KrishnaKTechnocredits) [inline src/aavrutiDJun20/MethodsOfCharacter.java:25]: Good job. Now it's perfect logic.

**Review by KrishnaKTechnocredits — APPROVED** [review]

**Pull request merged**